src/core/sagas/addons.js

```diff
@@ -0,0 +1,40 @@
+/* @flow */
+/* global Generator */
+import { hideLoading, showLoading } from 'react-redux-loading-bar';
+import { call, put, select, takeEvery } from 'redux-saga/effects';
+
+import { loadEntities } from 'core/actions';
+import { fetchAddon as fetchAddonFromApi } from 'core/api';
+import { FETCH_ADDON } from 'core/constants';
+import { ErrorHandler } from 'core/errorHandler';
+import log from 'core/logger';
+import type { FetchAddonAction } from 'core/actions/addons';
+
+import { getApi } from './utils';
+
+
+export function* fetchAddon(
```

Comment:
I'm only commenting here to say this code is very easy to follow and is making me happy we're moving to sagas. It makes our async code feel a bit more idiomatic too, compared to the other stuff which was a bit less consistently structured. 👍